app/liquid_tags/unified_embed/tag.rb

```diff
@@ -21,22 +21,16 @@ class Tag < LiquidTagBase
     # @return [LiquidTagBase]
     def self.new(tag_name, link, parse_context)
       klass = UnifiedEmbed::Registry.find_liquid_tag_for(link: link)
+      # If we don't know how to handle the embed, raise an error.
```

Comment:
The comment you're providing appears to reference the past behavior, which might make the comment less useful.

Consider the following:

```ruby
# If we can't find a registered "embed" tag, let's raise an exception.
# This exception will give the user an opportunity to adjust their approach.
#
# In a prior implementation, we chose to render an A-tag using the given URL.
# With that prior implementation, a user expecting a "rich embed" might not
# notice that they didn't have a rich embed and instead published a basic 
# A-tag.
```